core/haxlib.vcxproj

```diff
@@ -135,6 +135,7 @@
   <!-- Needed by any VcxProj -->
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <ImportGroup Label="ExtensionSettings">
+    <Import Project="$(VCTargetsPath)\BuildCustomizations\nasm.props" />

```

Comment:
NASM requires manually installing: https://github.com/ShiftMediaProject/VSNASM on VS2017.

This is somewhat annoying. Although there is a NASM package for NuGet, unfortunately its build customizations that do not work on Visual Studio 2017. I will consider updating that package, so that we don't have to deal with 3 separate sources (vcpkg, nuget, manual installation) for build-time dependencies.

Reply:
I see. We can live with the current approach, but it would be helpful if you could summarize the additional environment setup steps required to build and test this PR.